chunks/modelfreemodels.ipynb

```diff
@@ -395,11 +419,41 @@
     }
    },
    "source": [
-    "### Gaussian Processes\n",
+    "### Gaussian Processes in Data Analysis\n",
+    "\n",
+    "A draw from $P[y(x)|x^*]$ would represent a prior prediction for the function value $y(x^*)$\n",
```

Comment:
Yes, that does look better. I was trying to make a link to the notation used in Ivezic et al but how you have it works better I think.